pyls/plugins/jedi_completion.py

```diff
@@ -50,17 +50,11 @@
 
 @hookimpl
 def pyls_completions(config, document, position):
-    try:
-        definitions = document.jedi_script(position).completions()
-    except AttributeError as e:
-        if 'CompiledObject' in str(e):
-            # Needed to handle missing CompiledObject attribute
-            # 'sub_modules_dict'
-            definitions = None
-        else:
-            raise e
```

Comment:
Why did you remove this try/except?

Reply:
Already answered. https://github.com/palantir/python-language-server/pull/781#discussion_r422069504